handler/bot_test.go

```diff
@@ -0,0 +1,155 @@
+package handler
+
+import (
+	"testing"
+	"time"
+
+	"github.com/ichiro-its/discord-pr-bot/config"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+// MockGithubService is a mock implementation of service.GithubService for testing purposes
+type MockGithubService struct {
```

Comment:
(optional) I think it's better to write the mock services in separate files

Reply:
updated